### DISCUSSION TITLE
Icon in the toolbar to toggle between view only and remote control

### DISCUSSION THREAD

**Discussion by fernandeusto** [rustdesk/rustdesk#3639] [General]
### Description

<img width="214" alt="image" src="https://user-images.githubusercontent.com/87156183/224867510-0036b65f-c51a-4178-a2f1-ca0ec27ee691.png">
add an icon to the toolbar, to toggle between view only and remote control, and add in the client configuration an option to establish if the default is view only or remote control

### Impact

This allows you to connect to a client just to monitor that it is ok, and have a quick icon to be able to take control without the client having to enable or not the keyboard/mouse

**Comment by rustdesk** [comment]
#2755
